tests/Microsoft.Bot.Builder.TestBot.WebApi/Bots/DialogBot.cs

```diff
@@ -15,12 +15,15 @@ namespace Microsoft.BotBuilderSamples
     // each with dependency on distinct IBot types, this way ASP Dependency Injection can glue everything together without ambiguity.
     // The ConversationState is used by the Dialog system. The UserState isn't, however, it might have been used in a Dialog implementation,
     // and the requirement is that all BotState objects are saved at the end of a turn.
-    public class DialogBot<T> : ActivityHandler where T : Dialog 
+    public class DialogBot<T> : ActivityHandler
+        where T : Dialog
     {
+#pragma warning disable SA1401 // Fields should be private
         protected readonly Dialog _dialog;
         protected readonly BotState _conversationState;
         protected readonly BotState _userState;
         protected readonly ILogger _logger;
+#pragma warning restore SA1401 // Fields should be private
```

Comment:
This code is an exact drop of the core bot sample (in this case the webapi sample)

It would be great if we can maintain the consistency, that makes updates much easier. So can we exclude this warning outside of the code in an external rules file. That is possible with stylecop isn't it?